lib/src/infinite_list.dart

```diff
@@ -160,13 +160,15 @@ class InfiniteList extends StatefulWidget {
   final ItemBuilder itemBuilder;
 
   @override
-  _InfiniteListState createState() => _InfiniteListState();
+  InfiniteListState createState() => InfiniteListState();
```

Comment:
Can't this be `State<_InfiniteListState>`?

Reply:
Holy heck... 😂